framework/src/main/java/dev/ikm/komet/framework/observable/ObservableEntity.java

```diff
@@ -149,7 +146,7 @@ private synchronized static void updateVersions(Entity<? extends EntityVersion>
         boolean updateEntityReference = false;
```

Comment:
Does this need to be `synchronized` or even `static`?
Since it is `private` it is called internally.